packages/core/src/modules/ledger/ledgerUtil.ts

```diff
@@ -0,0 +1,5 @@
+import type * as Indy from 'indy-sdk'
+
+export function isLedgerRejectResponse(response: Indy.LedgerResponse): response is Indy.LedgerRejectResponse {
```

Comment:
We could do that, but `response is Indy.LedgerRejectResponse` is specifically meant for type casting the whole object. I think in your case it won't change the actual type, while this method does which provides better type hinting later on (as TS also knows it can't be a reject response anymore)